depthai_sdk/src/depthai_sdk/visualize/polygon.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC
+from typing import List
+
+
+class Vertex(ABC):
+    coords: List[..., float]
+
+
+class Vertex2D(Vertex):
```

Comment:
are these objects copying foxglove's mcap format, so we would be able to cross-use them?

Reply:
I didn't intend to copy anything, so those are just "placeholders" for the future functionality. If you think we can benefit from some specific format, I think we should add it in the future.

Reply:
I think we can take inspiration from their work, and if we can serialize these msgs directly and send to foxglove studio then that's even better (instead of needing to write a bridge), but that's just a nice-to-have